src/IBI.WZDx/Models/RoadEvents/RoadEventCoreDetails.cs

```diff
@@ -4,6 +4,8 @@
 using IBI.WZDx.Equality;
 using IBI.WZDx.Models.RoadEvents.WorkZones;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
is this needed for a specific line? Consider putting the warning disable line right before the line it is needed for, then enabling after.

Reply:
This comment applies in WorkZoneRoadEVent.cs as well.